django/db/backends/postgresql/compiler.py

```diff
@@ -1,10 +1,10 @@
-from django.db.models.sql.compiler import (
+from django.db.models.sql.compiler import (  # isort:skip
     SQLAggregateCompiler,
     SQLCompiler,
     SQLDeleteCompiler,
+    SQLUpdateCompiler,
```

Comment:
(I imagine this is where this import first was before isort tried to alphabetize it.)